app/controllers/pizzas_controller.rb

```diff
@@ -0,0 +1,28 @@
+class PizzasController < ApplicationController
+  def show
+    @pizzeria = Pizzeria.find(params[:pizzeria_id])
```

Comment:
Since you're doing this same thing in each controller action, you can use `before_action` to refactor. Not needed, but just an idea.